content/scalers/aws-sqs.md

```diff
@@ -35,6 +35,10 @@ triggers:
 
 You can use `TriggerAuthentication` CRD to configure the authenticate by providing either a role ARN or a set of IAM credentials.
 
+**Pod identity based authentication:**
+
+- `podIdentity.provider` needs to bet set on the `TriggerAuthentication` to either `kiam` or `eks` and the pod/service account must be configured correctly for your pod identity provider.
```

Comment:
Looks good to me, maybe just add a link to the full AWS pod identity docs?

Reply:
There are links to specific implementations in the corresponding blocks.